components/function-controller/internal/controllers/serverless/runtime/runtime.go

```diff
@@ -0,0 +1,73 @@
+package runtime
+
+import (
+	serverlessv1alpha1 "github.com/kyma-project/kyma/components/function-controller/pkg/apis/serverless/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+type Runtime interface {
+	SanitizeDependencies(dependencies string) string
+}
+
+type Config struct {
+	Runtime                 serverlessv1alpha1.Runtime
+	DependencyFile          string
+	FunctionFile            string
+	DockerfileConfigMapName string
+	RuntimeEnvs             []corev1.EnvVar
+}
+
+func GetRuntimeConfig(r serverlessv1alpha1.Runtime) Config {
+	switch r {
+	case serverlessv1alpha1.Nodejs12:
+		return Config{
+			Runtime:                 serverlessv1alpha1.Nodejs12,
+			DependencyFile:          "package.json",
+			FunctionFile:            "handler.js",
+			DockerfileConfigMapName: "dockerfile-nodejs-12",
+			RuntimeEnvs: []corev1.EnvVar{{Name: "NODE_PATH", Value: "$(KUBELESS_INSTALL_VOLUME)/node_modules"},
+				{Name: "FUNC_RUNTIME", Value: "nodejs12"},
+			},
+		}
+	case serverlessv1alpha1.Nodejs10:
+		return Config{
+			Runtime:                 serverlessv1alpha1.Nodejs10,
+			DependencyFile:          "package.json",
+			FunctionFile:            "handler.js",
+			DockerfileConfigMapName: "dockerfile-nodejs-10",
+			RuntimeEnvs: []corev1.EnvVar{{Name: "NODE_PATH", Value: "$(KUBELESS_INSTALL_VOLUME)/node_modules"},
+				{Name: "FUNC_RUNTIME", Value: "nodejs10"},
+			},
+		}
+	case serverlessv1alpha1.Python37:
+		return Config{
+			Runtime:                 serverlessv1alpha1.Python37,
+			DependencyFile:          "requirements.txt",
+			FunctionFile:            "handler.py",
+			DockerfileConfigMapName: "dockerfile-python-37",
+			RuntimeEnvs: []corev1.EnvVar{{Name: "PYTHONPATH", Value: "$(KUBELESS_INSTALL_VOLUME)/lib.python3.7/site-packages:$(KUBELESS_INSTALL_VOLUME)"},
```

Comment:
I would change the name from `PYTHONPATH` to `PYTHON_PATH`

Reply:
https://github.com/kubeless/runtimes/blob/master/stable/python/python.jsonnet#L45 unfortunately it has to be set to that value. I'll add that link as comment